src/main/java/com/tourism/psk/model/response/UserResponse.java

```diff
@@ -0,0 +1,63 @@
+package com.tourism.psk.model.response;
```

Comment:
I created a separate class for the response because I needed to return less information than the entity contains (response excludes username and password fields), but perhaps there is a "cleaner" way to do that?

Reply:
why is there a need for a response? void is good enough for post method

Reply:
It is specified in swagger that user details should be sent back after logging in, since client only provides the username and password.

Reply:
Will leave it as it is for now and merge so master branch is up to date for further work. If there is a need for it maybe will change the implementation later.